<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNew</class>
 <widget class="QDialog" name="DialogNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_Description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>511</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ImgSelect">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Selectioner une image</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Valid">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Validé</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Types :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxType"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>surface habitable :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxsSquareMeter"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>surface total :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxsSquareMeter_terrain"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nombre de pièces :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxNumberRoom"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>112</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Numéro et voie :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Code postal :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ville :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLineEdit" name="lineEdit_Adress"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ZipCode"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_City"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>130</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Adresse :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
